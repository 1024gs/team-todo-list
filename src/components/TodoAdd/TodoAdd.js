import React from 'react';
import './TodoAdd.scss';

const TodoAdd = ({onAdd}) => {
  return (
    <div className="TodoAdd">
      <div className="row">
        <div className="col s12">
          <button className="btn-flat waves-effect" onClick={onAdd}>
            <i className="material-icons left">add_circle</i>
            Add a to-do
          </button>
        </div>
      </div>
    </div>
  );
};

export default TodoAdd;
