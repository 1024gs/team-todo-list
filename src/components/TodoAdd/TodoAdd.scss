.TodoAdd {
  min-height: 80px;
  background-color: #f5f5f5;
  padding-top: 15px;
}

.TodoAdd button {
  padding-right: 9px;
  padding-left: 9px;
}
