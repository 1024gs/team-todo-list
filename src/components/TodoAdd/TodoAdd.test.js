import React from 'react';
import TodoAdd from './TodoAdd';
import sinon from 'sinon';
import {shallow} from 'enzyme';

const setup = (propOverrides) => {
  const props = Object.assign({onAdd: sinon.spy()}, propOverrides);
  const wrapper = shallow(<TodoAdd {...props} />);

  return {
    props,
    wrapper
  };
};

describe('TodoAdd', () => {
  it('renders', () => {
    const {wrapper} = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
