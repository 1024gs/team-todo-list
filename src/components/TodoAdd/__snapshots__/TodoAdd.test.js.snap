// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoAdd renders 1`] = `
<div
  className="TodoAdd"
>
  <div
    className="row"
  >
    <div
      className="col s12"
    >
      <button
        className="btn-flat waves-effect"
        onClick={[Function]}
      >
        <i
          className="material-icons left"
        >
          add_circle
        </i>
        Add a to-do
      </button>
    </div>
  </div>
</div>
`;
